### ISSUE TITLE
Numerical instabilities in "isobarAmplitude::gjTransform"

### ISSUE THREAD

**Issue by suhlatwork** [ROOTPWA-Maintainers/ROOTPWA#91]
"isobarAmplitude::gjTransform" never checks whether the beam and X Lorentz-vectors actually span a production plane or are collinear.

A proper solution probably has to handle two cases:
* beam and X are collinear and the three-vector component is not equal to zero
* beam and X are collinear with a zero three-vector component
Case one can probably not be handled without spoiling theta_GJ and phi_TY, the second case can probably be handled by simply not performing any transformation (as X already appears to be in its centre-of-mass system) and relying on the user to have taken care that those two angles still are correct. However, the check for equal to zero needs to include some numerical margin, as typically X is calculated from its daughters.

This is actually a real problem, as the "phaseSpaceIntegral" class uses a non-interaction vertex which can only provide equal beam and X Lorentz-vectors to calculate the amplitudes.

**Comment by bgrube** [comment]
Oops, but the how does phaseSpaceIntegral handle the t' dependence?

**Comment by suhlatwork** [comment]
For the pure phaseSpaceIntegrals (without taking acceptance into account) there should be no t' dependence?

**Comment by bgrube** [comment]
Yes, I was confusing it with the phase-space integral we use for the likelihood.
